tests/integration/test_fim/test_registry/test_registry_wildcards/test_registry_wildcards.py

```diff
@@ -0,0 +1,301 @@
+
+    # Delete the created key and check it's deletion is detected
+    delete_registry(registry_parser[WINDOWS_HKEY_LOCAL_MACHINE], subkey, KEY_WOW64_64KEY)
+    event_deleted = check_registry_crud_event(callback=callback_key_event, path=path, type='deleted', timeout=T_10,
+                                              arch='x64')
+    assert event_deleted is not None, 'Did not find the expected "registry_key deleted" event'
+
+
+@pytest.mark.parametrize('configuration, metadata', zip(t2_configurations, t2_configuration_metadata), ids=t2_case_ids)
+def test_registry_value_wildcards(configuration, metadata, set_wazuh_configuration,
+                                  configure_local_internal_options_function, restart_syscheck_function,
+                                  wait_syscheck_start):
+    '''
+    description: Check the behavior of FIM when using wildcards to configure the path of registry keys, and validate
+                 when values are created inside a monitored key, creation, modification and deletion is detected
+                 correctly.
+
+    wazuh_min_version: 4.5.0
+
+    test_phases:
+        - setup:
+            - Set wazuh configuration.
+            - Clean logs files and restart wazuh to apply the configuration.
+        - test:
+            - Check that one or more keys are detected when the configured wildcard is expanded
+            - Create a registry_value inside the first monitored key and check
+            - Wait for scan and check registry_value has been detected as 'added'
+            - Modify the registry_value
+            - Wait for scan and check registry_value has been detected as 'modified'
+            - Delete the registry_value
+            - Wait for scan and check registry_value has been detected as 'deleted'
+        - teardown:
+            - Restore configuration
+            - Stop wazuh
+
+    tier: 1
+
+    parameters:
+        - configuration:
+            type: dict
+            brief: Configuration values for ossec.conf.
+        - metadata:
+            type: dict
+            brief: Test case data.
+        - set_wazuh_configuration:
+            type: fixture
+            brief: Set ossec.conf configuration.
+        - configure_local_internal_options_function:
+            type: fixture
+            brief: Set local_internal_options configuration.
+        - policies_file:
+            type: string
+            brief: path for audit policies file to use on restore_win_whodata_policies fixture
+        - restore_win_whodata_policies
+            type: fixture
+            brief: restores windows audit policies using a given csv file after yield
+        - restart_syscheck_function:
+            type: fixture
+            brief: restart syscheckd daemon, and truncate the ossec.log.
+        - wait_for_fim_start_function:
+            type: fixture
+            brief: check that the starting fim scan is detected.
```

Comment:
Please, add the correct parameters..

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/4220/commits/cc756e243308370882f0aab65b88291a92535154